python/arcticdb/version_store/_store.py

```diff
@@ -959,14 +959,14 @@ def batch_read(
         return {v.symbol: v for v in versioned_items}
 
     def _batch_read_to_versioned_items(
-        self, symbols, as_ofs, date_ranges, columns, query_builder, throw_on_missing_version, kwargs=None
+        self, symbols, as_ofs, date_ranges, columns, query_builder, throw_on_error, kwargs=None
     ):
         if kwargs is None:
             kwargs = dict()
         version_queries = self._get_version_queries(len(symbols), as_ofs, **kwargs)
         read_queries = self._get_read_queries(len(symbols), date_ranges, columns, query_builder)
         read_options = self._get_read_options(**kwargs)
-        read_options.set_batch_throw_on_missing_version(throw_on_missing_version)
+        read_options.set_batch_throw_on_error(throw_on_error)
```

Comment:
Is there a reason why it didn't go in _get_read_options?

Reply:
I hadn't thought of it to be honest. It isn't a `kwarg` though, so we'd have to modify them in the calling functions, which feels like a bit of an odd pattern?